.circleci/config.yml

```diff
@@ -36,7 +36,7 @@ jobs:
             else
               echo "Building new docker images"
               docker-compose build
-              docker save -o ~/docker-cache.tar h2o python
+              docker save -o ~/docker-cache.tar python
```

Comment:
@bensteinberg Do you know what this should be? `h2o-python python postgres`?

Reply:
just `python`, I think -- only the containers we build

Reply:
Ah, no, sorry -- `web`?

Reply:
Thanks! If [`docker save`](https://docs.docker.com/engine/reference/commandline/save/) takes image names as args, and if we only want only the ones we build, then I think `h2o-python` is correct... which we can rename after a time, once the ruby is long gone.